airbyte-workers/src/main/java/io/airbyte/workers/temporal/scheduling/ConnectionManagerWorkflowImpl.java

```diff
@@ -257,6 +257,7 @@ private CancellationScope generateSyncWorkflowRunnable(final ConnectionUpdaterIn
         // silently ignore it.
         if (childWorkflowFailure.getCause() instanceof CanceledFailure) {
           // do nothing, cancellation handled by cancellationScope
+          log.info("");
         } else if (childWorkflowFailure.getCause()instanceof final ActivityFailure af) {
```

Comment:
This is existing code, but there appears to be a formatting issue here, there should be a space before `instanceof`

Reply:
Fixed in https://github.com/airbytehq/airbyte/pull/16454